src/pystatis/http_helper.py

```diff
@@ -92,7 +92,18 @@ def get_data_from_endpoint(
     Returns:
         requests.Response: the response object holding the response from calling the Destatis endpoint.
     """
-    db_host, db_user, db_pw = db.get_db_settings()
+    
+    # Determine database by matching regex to item code
+    name = params.get("name", params.get("selection", ""))
+    use_db = ""
+
+    if name is not None:
+        use_db = db.match_db(name)
+    
+    if not use_db:
+        use_db = _get_db_from_user_input()
```

Comment:
we should not use this here as this would mean in the worst case the method can only be used if the user enters something in the console because we have a `input` call inside this function. Instead, this function should just fail. However, I am not sure how this works out for Find endpoints and things like that where we need to get data but don't have a name/number so I guess we have to think more about this use case

Reply:
I already implemented my idea of a workaround or solution for now!